test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3240,6 +3240,28 @@ def forward(self, input):
         x = torch.randn(5, 4, 3)
         self.run_test(SplitModel2(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_chunk(self):
+        class ChunkModel(torch.nn.Module):
+            def __init__(self):
+                super(ChunkModel, self).__init__()
+
+            def forward(self, x):
+                return torch.chunk(x, 3, dim=1)
+
+        model = ChunkModel()
+        model.eval()
+        f = io.BytesIO()
+        x = torch.randn(1, 18)
+
+        ort_sess = convert_to_onnx(model, input=(x,), opset_version=self.opset_version, input_names=['x'],
+                                   dynamic_axes={'x': {0: 'batch_size', 1: 'dims'}})
+        for dim_size_ in range(13, 16):
+            y = torch.randn(1, dim_size_)
```

Comment:
why not use `self.run_test` for testing? If you wish to test with additional data, there is an argument `test_with_inputs` in `self.run_test` you can use.

Reply:
done